jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -84,32 +86,33 @@ protected void startNotificationThread() {
 				return thread;
 			};
 
-			Thread notifier = AccessController.doPrivileged(createThread);
+			Thread notifier = java.security.AccessController.doPrivileged(createThread);
 			notifier.start();
 		}
 	}
 
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
 	 */
-	/*[IF Sidecar19-SE]
+	/*[IF JAVA_SPEC_VERSION > 8]*/
 	@Deprecated(forRemoval = true, since = "1.8")
-	/*[ELSE]*/
+	@SuppressWarnings("deprecation")
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
 	@Deprecated
-	/*[ENDIF]*/
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemory() {
-		return osinstance.getTotalPhysicalMemorySize();
+		return getTotalPhysicalMemorySize();
 	}
 
+	/*[IF JAVA_SPEC_VERSION > 8]*/
+	@SuppressWarnings("deprecation")
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemorySize() {
+		/*[IF JAVA_SPEC_VERSION >= 14]*/
+		return osinstance.getTotalMemorySize();
+		/*[ELSE] JAVA_SPEC_VERSION >= 14 */
 		return osinstance.getTotalPhysicalMemorySize();
-	}
-
-	/**
-	 * TODO Was this intended to be exposed in com.ibm.lang.management.MemoryMXBean?
-	 */
-	public long getUsedPhysicalMemory() {
```

Comment:
The argument for removing this is valid for Java 11, but this can be used in Java 8 by casting to the implementation class. Even for Java 11 command line options can allow this to be used. On second thought I think we should play it safe and deprecate this first, remove it later.

Reply:
Agreed: I will update accordingly.